src/sas/qtgui/Utilities/Preferences/PreferencesWidget.py

```diff
@@ -34,15 +49,15 @@ def cb_replace_all_items_with_new(cb: QComboBox, new_items: List[str], default_i
     cb.setCurrentIndex(index)
 
 
-def config_value_setter_generator(attr: str,dtype: Optional[Any] = None):
+def config_value_setter_generator(attr: str, dtype: Optional[Any] = None):
```

Comment:
```suggestion
def config_value_setter_generator(attr: str, dtype: Optional[Callable] = None):
```
to match signature above